whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -2841,6 +2841,7 @@ class PostgreSQLComponent {
         return pool
     }
 
+    // FIXME: de-KBV/Libris-ify: maybe
```

Comment:
Not first priority?
Fix this when improving provenance handling

Reply:
Yes, agreed.